web/app/js/components/ExpandableTable.jsx

```diff
@@ -21,6 +21,9 @@ const styles = theme => ({
     marginTop: theme.spacing.unit * 3,
     overflowX: 'auto',
   },
+  expandedWrap: {
+    wordBreak: `break-word`
```

Comment:
Mostly for my own understanding... I noticed that `break-word` is supposedly not compatible with Firefox and IE:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#Browser_compatibility

...but I guess through the magic of, react?, it removes that property and correctly breaks the word in Firefox:
![firefox](https://user-images.githubusercontent.com/236915/54552913-5ef73400-496e-11e9-9296-f3c319259e02.png)

... and funny enough, with Chrome supporting `break-word`, it does a slightly worse job of breaking the string:
![chrome](https://user-images.githubusercontent.com/236915/54552897-5868bc80-496e-11e9-9ad5-4f86c0340538.png)

Reply:
@siggy, great thoughts/questions - my understanding is that typically, Material-UI [adds its own magic to unsupported browser features](https://material-ui.com/getting-started/supported-platforms/), but in this case, the word-wrap for this specific scenario was only broken in Chrome (not Firefox) so no extra CSS was needed to make it FF-compatible. (Pssst, automated cross-browser testing is in our future which will help with these comparisons. 🔮🤞) 

BUT, in digging into the CSS to figure out how Material-UI was handling Firefox, @rmars and I spotted some other layout problems and saw some opportunities to improve this modal - so thanks for the questions! Merging this PR now and reworking the modal in a separate PR.